internal/operators/odf/odf_operator.go

```diff
@@ -101,6 +105,31 @@ func getODFDeploymentMode(numOfHosts int) odfDeploymentMode {
 	return standardMode
 }
 
+func (o *operator) GetStorageClassName() string {
+	return defaultStorageClassName
+}
+
+func (o *operator) getMinDiskSizeGB(cluster *models.Cluster) int64 {
+	minDiskSizeGB := o.config.ODFMinDiskSizeGB
+	if common.HasOperator(cluster, mce.Operator.Name) {
+		minDiskSizeGB += mce.GetMinDiskSizeGB(cluster)
+	}
+	return minDiskSizeGB
+}
```

Comment:
This function seems common to both storage operators 🤔 could we potentially have this function defined once and use it in both?

Reply:
Good point, I've also extracted mce operator dependency and done this operation from the manager